src/main/scala/algolia/dsl/GetDsl.scala

```diff
@@ -89,6 +94,17 @@ trait GetDsl {
     def dictionarySettings: GetSettingsDictionaryDefinition =
       GetSettingsDictionaryDefinition()
 
+    def recommendation(
```

Comment:
these should be plural as the response is an array

Reply:
Yup, nice catch! I updated it